### DISCUSSION TITLE
worlflow don't update source archive

### DISCUSSION THREAD

**Discussion by surfzoid** [community/community#45028] [Actions]
### Select Topic Area

Bug

### Body

Hi
Tonight, i can erase my last release, commit some code, but at the end of actions, when release is created again, the 2 archive of source code are lay an old version, from 5 hours ago.
to easily check that go to release page https://github.com/surfzoid/QtVsPlayer/releases/tag/1.0.33  open first an archive, lets say, the tar.gz one, browse lib/aarch64 you will see 4 lib, now open a linux package, open it as an archive, you will see 3 lib, you can also look at the content of the .pro file

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @surfzoid ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by surfzoid** [reply]
Thank yu @queenofcorgis more infos about my problem:
i'm using workflow to automatically build pkg, i'm facing to a problem, few hours later i see i made a minor error, like a typo error, i modify my code and then commit/push, so workflow build again and update my binary package but not the zip and tarball source code in release, as we can see here https://github.com/surfzoid/QtVsPlayer/releases/tag/1.0.35

1 hour between both
because i'm using tar.gz in my spec file, source code isn't up to date, is there an mark workflow command to force the update or really  delete the tarball?
